src/boot/ksdk1.1.0/devAS7262.c

```diff
@@ -188,7 +172,39 @@ readSensorRegisterAS7262(uint8_t deviceRegister)
 	}
 
 
+	if (returnValue == kStatus_I2C_Success)
+	{
+		//...
+	}
+	else
+	{
+		#ifdef AS7262_DGBUG		
+		SEGGER_RTT_WriteString(0, "\t\t AS7262 Driver : comm fail");
+		#endif
+
+		return kWarpStatusDeviceCommunicationFailed;
+	}
+
+	return kWarpStatusOK;
+}
+
+
+WarpStatus
+LEDonAS7262(void) {
```

Comment:
Please see the coding conventions at https://github.com/physical-computation/Warp-firmware/blob/master/doc/README-CodingConventions.md